Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
```

Comment:
### [airbrake](https://airbrake.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 11.0.1 → 11.0.3

[_~~change-log~~, [source-code](https://github.com/airbrake/airbrake), [gem-diff](https://my.diffend.io/gems/airbrake/11.0.1/11.0.3)_]





<details>
<summary>Commits</summary>

A change of **32** commits. See the full changes on [the compare page](https://github.com/airbrake/airbrake/compare/v11.0.1...v11.0.3).

These are the first 10 commits:
- (593b0d1) [README: touch up, update outdated & incorrect info](https://github.com/airbrake/airbrake/commit/593b0d18bb2d724e4fcb8fe88b34826ac42ce9f1)
- (72391a6) [Merge pull request #1135 from airbrake/readme-touch-up](https://github.com/airbrake/airbrake/commit/72391a623dfcf35144594815ce3ea61591d6ee73)
- (94bdabf) [circleci: add Dockerhub authentication](https://github.com/airbrake/airbrake/commit/94bdabf761ebb4a91b98a905175da9eecc01bb6c)
- (100d3de) [Merge pull request #1137 from airbrake/circleci-dockerhub-au…](https://github.com/airbrake/airbrake/commit/100d3de4ad3b2d049bb51f36bdb922f232ee5c99)
- (f0a15c0) [README: update Airbrake logo](https://github.com/airbrake/airbrake/commit/f0a15c0b23e8b5a9a23e173354f9d51752b5c2c3)
- (d2d23ea) [Merge pull request #1139 from airbrake/logo-update](https://github.com/airbrake/airbrake/commit/d2d23ea9d3509ee1dac7521cf202ea3db5a5bc72)
- (c3014ca) [circleci: verify that all YARD @since tags start with 'v'](https://github.com/airbrake/airbrake/commit/c3014cab92a406cae6cb85bcda32ecfb4b72fa48)
- (dfbd162) [Merge pull request #1140 from airbrake/since-version-lint](https://github.com/airbrake/airbrake/commit/dfbd162a6f9f6bdeee7eeb5e7d1449d1770091fb)
- (fa88afb) [gemspec: lock "ethon" to v0.12.0](https://github.com/airbrake/airbrake/commit/fa88afb76174227e5a7950a2c74f3fbfa75e98a3)
- (619cdf0) [Merge pull request #1153 from airbrake/build-fix](https://github.com/airbrake/airbrake/commit/619cdf0bc9605f3f79ed938faaf595ae060885d4)


</details>